test/solvers/trunkls.jl

```diff
@@ -0,0 +1,40 @@
+using Krylov
+using Printf
+
+@testset "Simple test" begin
+    model = ADNLSModel(x -> [10 * (x[2] - x[1]^2), 1 - x[1]], [-2.0, 1.0], 2)
+    for subsolver in Optimize.trunkls_allowed_subsolvers
+        stats = trunk(model, subsolver=subsolver)
+        @test stats.status == :first_order
+        isapprox(stats.solution, ones(2), rtol=1e-4)
+        @test isapprox(stats.objective, 0, atol=1e-6)
+        @test neval_jac_residual(model) == 0
+        stline = statsline(stats, [:objective, :dual_feas, :elapsed_time, :iter, :neval_residual,
+                                   :neval_jprod_residual, :neval_jtprod_residual, :status])
+        @info @sprintf("%8s  %5d  %5d  %s\n", model.meta.name, nlp.nls_meta.nvar, nlp.nls_meta.nequ, stline)
```

Comment:
`nlp` -> `model`
It looks like there are some silent errors when we call `@info` and its arguments produces errors: https://travis-ci.org/JuliaSmoothOptimizers/Optimize.jl/jobs/490333777#L875

Reply:
Good catch!

Reply:
Should be fixed. It's not good that those don't trigger a failure.